.flake8

```diff
@@ -2,5 +2,6 @@
 
 [flake8]
 # E265: block comment should start with '#'
-ignore = E265
+# F821: undefined name
+ignore = E265,F821
```

Comment:
I suggest using `extend-ignore` rather than `ignore` that way if someone passes command-line arguments these don't get overridden.